README.md

```diff
@@ -100,7 +100,7 @@ Cilium requires a recent version of the Linux kernel, iproute2 and clang+LLVM.
 All required changes have been merged upstream but may have not been included
```

Comment:
Can you rewrite this as well? it is outdated as all changed have been merged. Something like:
```
All required changes have been merged upstream and are available in official releases:
 * Linux >= 4.8.0: http://www.kernel.org/
 * iproute2 >= 4.8.0: https://www.kernel.org/pub/linux/utils/net/iproute2/
 * clang+LLVM >=3.7.1: http://llvm.org/releases

Cilium can make use of additional functionality available in >= 4.9 kernels. It will probe for the availability upon startup and enable it automatically. Development snapshots of the Linux kernel and iproute2 tree can be found here.
 * git tree
 * git tree
```